### PR TITLE
[16.0][MIG][FIX] mrp_analytic

### PR DESCRIPTION
**Pull request by Hardik-OSI**

### CODE REVIEW

File: mrp_analytic/tests/test_mrp_analytic.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    def test_num_productions(self):
+        self.assertEqual(self.analytic_account.num_productions, 1)
+
+    # def test_carry_to_move_line(self):
+    #     self.production.button_mark_done()
+    #     account_moves = self.env["account.move.line"].search(
+    #         [("analytic_account_id", "=", self.analytic_account.id)]
+    #     )
+    #     self.assertEqual(len(account_moves), 4)
```
Comment (dreispt) [inline mrp_analytic/tests/test_mrp_analytic.py:70]: @murtuzasaleh Can you reactivate this test case please?

**Comment by dreispt** [comment]
Please fix pre-commit checks

**Comment by dreispt** [comment]
Closes #494

**Comment by yostashiro** [comment]
There is this PR https://github.com/OCA/account-analytic/pull/536 for assigning analytic distribution on component moves. Does this module intends to cater to different business needs (especially the stock accounting part)?

Please see this comment https://github.com/OCA/account-analytic/pull/536#issuecomment-1433169835 as well.

**Pull request closed without merging**